sw/device/tests/clkmgr_reset_frequency_test.c

```diff
@@ -74,9 +75,12 @@ bool test_main(void) {
     CHECK_DIF_OK(dif_rstmgr_software_device_reset(&rstmgr));
   } else if (rstmgr_testutils_reset_info_any(&rstmgr, kDifRstmgrResetInfoSw)) {
     LOG_INFO("Back from rstmgr SW reset");
-    CHECK(clkmgr_testutils_check_measurement_enables(&clkmgr,
-                                                     kDifToggleDisabled));
-    CHECK(clkmgr_testutils_check_measurement_counts(&clkmgr));
+    bool all_disabled =
+        CHECK_STATUS_OK(clkmgr_testutils_check_measurement_enables(
```

Comment:
why is a `CHECK_STATUS_OK` being assigned to a boolean?

Reply:
I also changed the `CHECK_STATUS_OK` macro to unwrap the `status.value` just like the `TRY` macro.
https://github.com/lowRISC/opentitan/blob/04641452e05d96471a8762ddfb19b8a01546da82/sw/device/lib/testing/spi_flash_testutils.c#L144-L149

Please, let me know if you see any issue with that.

Reply:
Sorry where did you change this? I see it for TRY [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/base/status.h;drc=a14866dd639e1a8ce24c852adbf83a51f0760c03;l=114), but I don't see anything for `CHECK_STATUS_OK` [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_framework/check.h;drc=5ac26cd380616758f7919cc467439b6ccc7dcaa5;l=262)?

Reply:
It's in the same commit [here](https://github.com/lowRISC/opentitan/pull/17947/files#r1166451172)

Reply:
I followed Amaury and @cfrantz advise and created a new macro called `UNWRAP`.